jerry-core/jerry.c

```diff
@@ -591,6 +591,47 @@ jerry_get_array_index_value (jerry_object_t *array_obj_p, /**< array object */
 } /* jerry_get_array_index_value */
 
 /**
+ * Get length of an array object
+ *
+ * @return length of the given array
+ */
+uint32_t
+jerry_get_array_length (const jerry_object_t *object_p)
+{
+  jerry_assert_api_available ();
+
+  JERRY_ASSERT (jerry_is_array (object_p));
```

Comment:
I'm not sure how I feel about this kind of an assert in an API function. I think it would be better to handle the case when the argument is not an array, for example an early `return 0` or something like that (since we can't throw errors here without changing the return type).